### DISCUSSION TITLE
Shorthand `do` syntax

### DISCUSSION THREAD

**Discussion by JonathanStarup** [flix/flix#8513] [Ideas]
If im annoyed writing `do operation(...)` I want to implement a shortand
```
eff Eff {
  pub def op(): Void
}
mod Eff {
  pub def op(): t \ Eff = do op()
}
```
but oh no! Compiler now says that I have duplicate definitions 😢

### Solutions
- Remove the `do` keyword
- Allow the duplicate names; One always have to prefixed by `do` so they could both exist

**Comment by JonathanStarup** [comment]
I think i remember you saying that this is the plan @magnus-madsen

**Comment by magnus-madsen** [comment]
I am undecided. We are discussing it with the HCI people.
